src/App.js

```diff
@@ -14,6 +14,17 @@ function App() {
     const [favouriteItems, setFavouriteItems] = useState([]);
     const [basketItems, setBasketItems] = useState([]);
 
+    useEffect(() => {
+        let basketProducts = JSON.parse(
+            window.localStorage.getItem('basketProduct')
+        );
+        let favouriteProducts = JSON.parse(
+            window.localStorage.getItem('favouriteProduct')
+        );
```

Comment:
Why do you use `let` instead of `const`? You're not changing the values here.

Reply:
I changed